schema/schema-solaris.json

```diff
@@ -0,0 +1,36 @@
+{
+    "solaris": {
+        "description": "Solaris platform-specific configurations",
+        "id": "https://opencontainers.org/schema/bundle/solaris",
+        "type": "object",
+        "properties": {
+            "milestone": {
+                "id": "https://opencontainers.org/schema/bundle/solaris/milestone",
+                "type": "string"
+            },
+            "limitpriv": {
+                "id": "https://opencontainers.org/schema/bundle/solaris/limitpriv",
+                "type": "string"
+            },
+            "max-shm-memory": {
+                "id": "https://opencontainers.org/schema/bundle/solaris/max-shm-memory",
+                "type": "string"
+            },
+            "capped-cpu": {
+                "id": "https://opencontainers.org/schema/bundle/solaris/capped-cpu",
+                "$ref": "defs.json#/definitions/mapStringString"
```

Comment:
This is just the initial PR, we will expand(more properties to come) and make it more structured in the future. For now, I think this would be fine.